### PR TITLE
Add Jupyter 

### PR DESCRIPTION
**Pull request by jgmize**
- https://gitlab.com/mozmeao/snippets-service/-/jobs/372926034

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -29,4 +46,4 @@ services:
       - CSP_REPORT_ENABLE=False
```
Comment (glogiotatidis) [inline docker-compose.yml]: `web` has a bunch of env variables set originally and you only leave it with 3. Copy / paste erorr?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -17,6 +17,17 @@ build-images:
     - docker build -t ${DOCKER_IMAGE_TAG} --build-arg GIT_SHA=${CI_COMMIT_SHA} --pull .
     - docker push ${DOCKER_IMAGE_TAG}
 
+build-jupyter:
```
Comment (glogiotatidis) [inline .gitlab-ci.yml]: This  section seems disconnected from the rest of the file. It does things in a different way, e.g. using compose, not tagging the built images, not using the set variables.

File: docker-compose.yml (hunk 1 above)
Comment (jgmize) [reply docker-compose.yml]: good catch, thanks! I've restored the accidentally deleted env vars in ad0ff18

File: .gitlab-ci.yml (hunk 2 above)
Comment (jgmize) [reply .gitlab-ci.yml]: Good points, I've updated it in c09c0d1 to be more in alignment with the rest of the file.

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -16,6 +17,8 @@ build-images:
   script:
     - docker build -t ${DOCKER_IMAGE_TAG} --build-arg GIT_SHA=${CI_COMMIT_SHA} --pull .
     - docker push ${DOCKER_IMAGE_TAG}
+    - docker build -t ${JUPYTER_IMAGE_TAG} --build-arg DEVELOPMENT=true .
```
Comment (glogiotatidis) [inline .gitlab-ci.yml]: Let's keep this is separate step

File: docker-compose.yml (hunk 1 above)
Comment (glogiotatidis) [reply docker-compose.yml]: Thanks for updating. I belive `ENABLE_ADMIN` and the `CSP` variables are not needed to run Jupiter. Let's try to keep things tidy!

Comment (jgmize) [reply docker-compose.yml]: thanks, removed in b64932c

**Pull request merged**